mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -1594,6 +1594,73 @@ LogicalResult DataBoundsOp::verify() {
   return success();
 }
 
+LogicalResult PrivateClauseOp::verify() {
+  Type symType = getSymType();
+
+  auto verifyTerminator = [&](Operation *terminator) -> LogicalResult {
+    if (!terminator->hasSuccessors() && !llvm::isa<YieldOp>(terminator))
+      return mlir::emitError(terminator->getLoc())
```

Comment:
Is the mlir prefix here and in several places below required?

Reply:
Yes, because I don't want this to get resolved to `this->emitError(..)` in order to provide more accurate location information when needed.